docs/7.0-ProwCIIntegrationTest.md

```diff
@@ -0,0 +1,78 @@
+# 7.0 Prow CI Integration Test Framework
+
+Content:
+
+- [7.0 Prow CI Integration Test Framework](#70-prow-ci-integration-test-framework)
+  - [7.1 - CI Operator Tooling(Prow)](#71-ci-operator-toolingprow)
+  - [7.2 - CI Operator & AWS Account Operator](#72-ci-operator--aws-account-operator)
+    - [7.2.1 - Steps for updating AAO Prow configuration](#721-steps-for-updating-aao-prow-configuration)
+  - [7.3 - Integration test CI step configuration](#73-integration-test-prow-ci-step-configuration)
+  - [7.4 - Integration Test Script](#74-integration-test-script)
+    - [7.4.1 - Integration Test flow diagram](#741-integration-test-flow-diagram)
+    - [7.4.2 - Integration Test flow diagram](#742-artifacts-and-logs-of-test-steps)
+
+## 7.1 CI Operator Tooling(Prow)
+
+The [CI Operator](https://docs.ci.openshift.org/docs/architecture/ci-operator/) or Openshift CI is a set of tools designed for use with Github repositories present in [Openshift Organization](https://github.com/openshift). It helps to perform end-to-end test suites to focus on the content of their tests and hiding the compexity of assembling an infrastructure for openshift cluster setup and installation. Openshift CI uses pipelines, builds and image streams extensively to achieve e2e test step runs.
+
+## 7.2 CI Operator & AWS Account Operator
+
+![Openshift CI Flow Diagram](/docs/images/prow-CI.png)
+
+AWS Account Operator in particular uses Openshift CI to define and run Prow jobs when PRs are submitted, altered or merged. CI operator tooling documentation defines a way to setup CI flow for any repository.
+
+AWS Account Operator has an existing Prow configuration present [here](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/aws-account-operator/openshift-aws-account-operator-master.yaml).
+  
+### 7.2.1 Steps for updating AAO Prow configuration
+  - Perform changes in existing configuration [here](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/aws-account-operator/openshift-aws-account-operator-master.yaml)
+  - Execute `make update`(assure docker service is running). This `update` target will update [jobs files](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/aws-account-operator/openshift-aws-account-operator-master.yaml) according to changes done in above step.
+  - openshift/release PR runs a rehearsal step against AAO master as part of presubmit check using the updated prow configuration. Watch for step named - `ci/rehearse/openshift/aws-account-operator-master-integration-test`
+  - commit and merge changes to [release repository](https://github.com/openshift/release)
```

Comment:
```suggestion
For complete details on working with the Openshift release repository, refer to [their documentation](https://docs.ci.openshift.org/docs/how-tos/contributing-openshift-release/). The following steps will get you started for AAO's CI Operator use:

# Update [AAO Prow source file](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/aws-account-operator/openshift-aws-account-operator-master.yaml).
# Run `make-update` (ensure docker service is running). This will re-generate [AAO's Prow job definitions](https://github.com/openshift/release/tree/master/ci-operator/config/openshift/aws-account-operator)
# Commit all changes - both source file(s) and generated file(s).
# Submit a PR to [Openshift release repository](https://github.com/openshift/release). A rehearsal step will run against the AAO master branch as part of a presubmit check using the updated prow configuration (meaning all tests will be run just like they would if you submit a PR to the AAO repo).
```